iodhbwm.tex

```diff
@@ -261,13 +261,16 @@
                 Eine ausführliche Beschreibung befindet sich auf \url{https://tex.stackexchange.com/a/69484/142408}.
             
             \subsubsection{TeXlive}
-                \Warning{Die Installationsanweisung für TeXlive wurde nicht getestet!}
+                \Info{Manche dieser Schritte können villeicht übersprungen werden!}
                 
                 \begin{enumerate}
                     \item \verb|kpsewhich -var-value TEXMFHOME| zum Auffinden des Ordner
-                    \item ~/texmf/tex/latex/ (ggf. neu anlegen)
-                    \item  Ordner iodhbwm nach kopieren ~/texmf/tex/latex/ (Ordner muss zwingend iodhbwm heißen)
+                    \item ~/.texmf-config/tex/latex/ (ggf. neu anlegen)
```

Comment:
Can you provide a link or some documentation which says that custom packages are stored inside _.texmf-config_ folder. I can't find any documentation about this folder.

Reply:
As far as I can tell `~/texmf` is the default. Personally I much prefer to use a folder prefixed with a dot, so that it is hidden, but for install instructions one should stick to standards I think :)